app/javascript/mastodon/features/ui/components/sign_in_banner.jsx

```diff
@@ -33,7 +35,7 @@ const SignInBanner = () => {
     <div className='sign-in-banner'>
       <p><FormattedMessage id='sign_in_banner.text' defaultMessage='Sign in to follow profiles or hashtags, favourite, share and reply to posts. You can also interact from your account on a different server.' /></p>
       <a className='button button--block' href='/auth/auth/openid_connect' rel='nofollow' data-method='post'><FormattedMessage id='sign_in_banner.sign_in' defaultMessage='Sign in' /></a>
```

Comment:
We want a similar change here too ad point to `/auth/auth/openid_connect`. Something like:
```
      <form action='/auth/auth/openid_connect' method='post'>
        <button className='button button--block button-tertiary' type='submit'>
          <FormattedMessage id='sign_in_banner.sign_in' defaultMessage='Login' />
        </button>
      </form>
```

Reply:
@bakulf – Happy to add this, but want to better understand the goal.  Is it for consistency?  (Using `<form>` instead of the odd link with `data-method='post'`?)

I originally left the "sign-in" button as-is to avoid too many divergent changes on our fork.

Reply:
data-method is fine too.